### PR TITLE
[kube-parrot] add alerts

### PR DESCRIPTION
**Pull request by defo89**
`enable` switch will be in the umbrella chart

### CODE REVIEW

File: system/kube-proxy/values.yaml [hunk 1]
```diff
@@ -35,3 +35,10 @@ parrot:
 
 externalip: {}
 #  ignoreaddress: 
+
+# Deploy kube-proxy Prometheus alerts.
+alerts: {}
+  # enabled: true
```
Comment (auhlig) [inline system/kube-proxy/values.yaml]: I'd enable that by default.

File: system/kube-proxy/templates/alerts.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{{- $values := .Values }}
+{{- if $values.alerts.enabled }}
+{{- if $values.sidecar.parrot.enabled }}
```
Comment (auhlig) [inline system/kube-proxy/templates/alerts.yaml]: i think it's `$values.sidecars.parrot`

Comment (defo89) [reply system/kube-proxy/templates/alerts.yaml]: tnx, you are right - fixed this

**Comment by auhlig** [comment]
🎉

**Pull request merged**